srcpkgs/reuse/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'reuse'
+pkgname=reuse
+version=0.13.0
+revision=1
+build_style=python3-module
+make_check_args="-k not((test_lint_read_errors)or(test_generate_project_report_read_error))"
+hostmakedepends="python3-setuptools python3-setuptools_scm gettext"
+depends="python3-binaryornot python3-boolean.py python3-debian python3-Jinja2
+ python3-license-expression python3-requests"
+checkdepends="python3-pytest $depends"
+short_desc="Helper tool for compliance with REUSE Specification"
+maintainer="k4leg <python.bogdan@gmail.com>"
+license="GPL-3.0-or-later, CC-BY-SA-4.0, CC0-1.0, Apache-2.0"
```

Comment:
All original source code is licensed under GPL-3.0-or-later.
    All documentation is licensed under CC-BY-SA-4.0.
    Some configuration and data files are licensed under CC0-1.0.
    Some code borrowed from spdx/tool-python is licensed under Apache-2.0.

I think we don't need to list CC* licenses here.

Reply:
Yes you are right. But the package still contains files with license CC0-1.0 (for example `/usr/lib/pythonX.Y/site-packages/reuse/resources/exceptions.json`), so I will only remove license CC-BY-SA-4.0.
Thanks!